appserver/tests/payara-samples/test-domain-setup/src/test/java/fish/payara/samples/setuptests/RestartDomainTest.java

```diff
@@ -71,61 +65,10 @@ public void resetLogging() throws Exception {
         Thread.sleep(50L);
     }
 
-    /**
```

Comment:
You have to enforce order of tests, otherwise this change makes the test unstable or even dead.

Reply:
it is alphabetically enforced